### PR TITLE
Add USSD base and registration

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
- [x] add registration base
- [x] add basic flow
- [x] add simple functionality; initial check of user registration status and save to hub/registration store

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review el'capitan

File: Gruntfile.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ module.exports = function (grunt) {
         paths: {
             src: {
                 app: {
-                    smsapp: 'src/smsapp.js'
+                    smsapp: 'src/smsapp.js',
+                    ussd_registration: 'src/ussd_registration.js'
```
Comment (imsickofmaps) [inline Gruntfile.js]: The USSD line is shared for registration and quizzes so lets just make this `ussdapp` and file names etc.

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps rename(s) done

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**